docs/scenarios/docker_ssi.md

```diff
@@ -58,7 +58,7 @@ There is only one scenario declared: "`DOCKER_SSI`". But this unique scenario ca
 To help us to run a concrete case of the matrix variants, there are two scripts to be executed locally:
 
 * `utils/build/ssi/build_local_manual.sh`: Runs the Docker SSI scenario using the arguments: library, weblog, docker base image, architecture and installable runtime. To run this command you need to know the correct combination of these arguments.
```

Comment:
This file was removed in this commit: https://github.com/DataDog/system-tests/commit/f7362a42f05b5c8b7c129c4d41830962524e08f8#diff-fd364120f0f474ec86a552d6e581ed6159b9b88674b41150ca18a25fd1af492d. We should also remove it from this doc.